src/extra/components/pulses.py

```diff
@@ -1507,6 +1507,59 @@ def pulse_mask(self, labelled=True, field=None):
         else:
             raise ValueError(f"{field=!r} parameter was not 'fel'/'ppl'/None")
 
+    def pumped_pulses_ratios(self, ppl_only_value=np.nan, labelled=True):
```

Comment:
`labelled` is never used. I don't have a strong preference whether we implement it or remove it.

Reply:
Well spotted. I realized at the end all other methods carry it, but forgot the implementation...